### ISSUE TITLE
Bump search max result window (to 30000

### ISSUE THREAD

**Issue by diox** [mozilla/addons-server#19794]
For performance/scalability reasons, we set max_result_window in elasticsearch to 25000 - if there more results for a given query, we don't allow returning them.

25000 was chosen back in the day because it was above the total number of public extensions available, but we're now past that number. Let's try bumping it to 30000, hopefully that should still work and allow us to see all public extensions through the API.

The setting is `ES_MAX_RESULT_WINDOW`.

**Comment by diox** [comment]
To check the value set for the index:
```
curl -s elasticsearch:9200/addons/_settings?pretty | grep max_result_window
```

To set it to 30000 if needed:
```
curl -H 'Content-type: application/json' -XPUT -d '{"index": {"max_result_window": 30000}}' elasticsearch:9200/addons/_settings
```
(Replace `elasticsearch:9200` with the right host:port and `addons` with the right alias name)

**Comment by diox** [comment]
(I updated dev & stage ES clusters)

**Comment by AlexandraMoga** [comment]
> I remembered that the sitemap was using the 25k results limit as well ( see #17362). Will the new 30k limit apply automatically to the sitemap or does it need some extra tweaking?

It seems to have been applied automatically to sitemap pages. 

Also, the new pagination is working on stage as well.

**Comment by diox** [comment]
Good point about the sitemap: yes, it should automatically follow that new limit.
